src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -530,9 +530,13 @@ def render_worker_node_specific(config, args):
     worker_name = config["worker_node"][0].split(".")[0]
     common_worker_labels = get_kube_labels_of_machine_name(config, worker_name)
     config = get_stat_of_sku(config)
+    default_worker_f2cp = ["kubernetes_common", "kubelet_worker", "nfs_client"]
     for sku in config["worker_sku_cnt"]:
-        config["kube_labels"] = common_worker_labels + ["sku={}".format(sku), "gpuType={}".format(
-            config.get("sku_mapping", {}).get(sku, {}).get("gpu-type", "None"))]
+        gpuType = config.get("sku_mapping", {}).get(sku, {}).get("gpu-type", "None")
```

Comment:
python variable should be snake-cased: `gpu_type`

Reply:
this is kind of legacy code. should we change all "gpuType" and "gpu-type" to "gpu_type"?

Reply:
no need. I was talking about the python variable, not the variable names in configuration.